dist/api_data.py

```diff
@@ -1314,6 +1314,23 @@ def __ge__(self, other):
         type='boolean'),
 ]
 
+
+cursor_bound_config = [
+    Config('action', 'set', r'''
+        configures whether this call into the api will set or clear range bound on the given cursor,
```

Comment:
This is where we should specify that the cursor bounds API is in beta and shouldn't be used.